app/main.py

```diff
@@ -2,0 +1,14 @@
+def copy_file(command: str) -> None:
+    if command.startswith("cm"):
+        command = command.replace("cm ", "")
+    else:
+        print("Unknown command")
+    list_of_files_name = command.split(" ")
+    if list_of_files_name[0] != list_of_files_name[1]:
```

Comment:
The function should do nothing if the source and destination filenames are the same. This condition is correctly implemented.